next.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+};
+
+module.exports = nextConfig;
```

Comment:
You seem to have created a `Next` app 🤔 this shows great knowledge but this needs to be a `React` app created with `create-react-app`, you can create your app by running this:
```
npx create-react-app my-app
```
If you already created the repo on GitHub and you cloned it, you can run this:
```
npx create-react-app .
```
Which means "Create a React app in the current directory" 😉

NB: This supposes of course that you have `node`, `npm`, and `create-react-app` installed. 🤓